environments/dev/backend_lb_config.yaml

```diff
@@ -11,25 +11,33 @@ gateways:
     # Toffee application
       - product: toffee
         component: recipe-backend
+        priority: 10
```

Comment:
what's this?

Reply:
This has to be set in the latest azurerm api for some reason. In previous versions this is set automatically but now it has to be done manually. Haven't yet figured out how to generate the figure dynamically so it doesn't need to be coded in the vars
https://github.com/hashicorp/terraform-provider-azurerm/issues/17079

Reply:
is there some way to get an index and then add 10?

Reply:
Matt sent me a code snippet that I'm going to look to adapt for this